content/posts/derisking-guide-update.md

```diff
@@ -12,7 +12,7 @@ excerpt: |
 image: 
 ---
 
-Today we launch [De-risking Government Technology Guide 2.0](https://guides.18f.gov/derisking-government-tech/), the first update of the 18F De-risking Guide since it was published in 2020. Find all the recommendations from the original guide’s two parts—the State Software Budgeting Handbook and the Federal Field Guide—now in one unified package. Plus, a new, in-depth section on vendor management! 
+Today we launch a new version of the [De-risking Government Technology Guide](https://guides.18f.gov/derisking-government-tech/), its first update since being published in 2020. Find all the recommendations from the original guide’s two parts—the State Software Budgeting Handbook and the Federal Field Guide—now in one unified package. Plus, a new, in-depth section on vendor management! 
```

Comment:
concern: There's evidence that this hasn't gone through plain-language review: it uses some passive voice, there are sentence fragments, etc. Is there a plan to do a plain language review or check this against the 18F style guide?

Reply:
It did go through a plain-language review, but some things always slip through the cracks.